pytest/tests/adversarial/fork_sync.py

```diff
@@ -8,91 +8,90 @@
 
 sys.path.append('lib')
 
-
 from cluster import start_cluster
```

Comment:
Looks great! I tried autopep8 before but it incorrectly moved sys.path.append('lib') after imports, yapf looks smart enough and don't have this problem
PS: i'd like to keep indent=4, as existing file uses that.